src/main/scala/com.socrata.tileserver/services/TileService.scala

```diff
@@ -77,7 +77,12 @@ case class TileService(client: CuratedServiceClient) extends SimpleResource {
   { resp: Response =>
     def createResponse(parsed: (JValue, Seq[FeatureJson])): HttpResponse = {
       val (jValue, features) = parsed
-      logger.debug(s"Underlying json: ${jValue.toString}")
+
+      // $COVERAGE-OFF$
+      if (logger.isDebugEnabled) {
```

Comment:
You wouldn't have to do this if you used scalalogging library, or even the parameterized interface to SLF4J:

    logger.debug("Underlying json: {}", jValue)

(actually sorry, that might only be efficient for logback, not log4j.   But scalalogging is safe anyhow.)